### PR TITLE
continentInfo and countryUtils -> nameUtils

### LINKED ISSUE #785: [Feature Req] Coordinates in /Continents
**Is your feature request related to a problem? Please describe.**
No.

**Describe the solution you'd like**
Add continent coordinates.

**Additional context**
```json
{
  "NorthAmerica": { "lat": 31.6768272, "long": -146.4707474 },
  "SouthAmerica": { "lat": -15.6551563, "long": -100.7484231 },
  "Asia": { "lat": 23.7027273, "long": 62.3750637 },
  "Europe": { "lat": 25.771324, "long": -35.6012256 },
  "Africa": { "lat": 1.7383867, "long": -16.3094636 },
  "Oceania": { "lat": -8.6599161, "long": 91.1469847 }
}
```

### PR DESCRIPTION
**Pull request by puf17640**
Closes #785

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -39,7 +39,7 @@
     "dotenv": "^8.2.0",
     "ejs": "3.0.2",
     "express": "^4.16.3",
-    "ioredis": "*",
+    "ioredis": "4.17.3",
```
Comment (ebwinters) [inline package.json:42]: Did you update packages on purpose?

File: utils/continents.js [hunk 2]
```diff
@@ -0,0 +1,9 @@
+/* eslint-disable max-len */
```
Comment (ebwinters) [inline utils/continents.js:1]: Probably don't need this line

File: package.json (hunk 1 above)
Comment (puf17640) [reply package.json:42]: oh not really, will revert

File: utils/continents.js (hunk 2 above)
Comment (puf17640) [reply utils/continents.js:1]: true

**Comment by Nekiono** [comment]
hope the package-lock.json isnt broken

**Pull request merged**